scipy/optimize/_optimize.py

```diff
@@ -2877,36 +2877,49 @@ def _minimize_scalar_golden(func, brack=None, args=(),
 
 def bracket(func, xa=0.0, xb=1.0, args=(), grow_limit=110.0, maxiter=1000):
     """
-    Bracket the minimum of the function.
+    Bracket the minimum of a function.
 
     Given a function and distinct initial points, search in the
     downhill direction (as defined by the initial points) and return
-    new points xa, xb, xc that bracket the minimum of the function
-    f(xa) > f(xb) < f(xc). It doesn't always mean that obtained
-    solution will satisfy xa<=x<=xb.
+    three points that bracket the minimum of the function.
 
     Parameters
     ----------
     func : callable f(x,*args)
         Objective function to minimize.
     xa, xb : float, optional
-        Bracketing interval. Defaults `xa` to 0.0, and `xb` to 1.0.
+        Initial points. Defaults `xa` to 0.0, and `xb` to 1.0.
+        A local minimum need not be contained within this interval.
     args : tuple, optional
         Additional arguments (if present), passed to `func`.
     grow_limit : float, optional
         Maximum grow limit.  Defaults to 110.0
     maxiter : int, optional
         Maximum number of iterations to perform. Defaults to 1000.
 
+    Raises
```

Comment:
Think this should be between returns and notes as per https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
Ugh I even looked at the standards to confirm; then I went and put it in the wrong place. I'll fix it.